src/script/components/manifest-options.ts

```diff
@@ -412,14 +413,37 @@ export class AppManifest extends LitElement {
   constructor() {
     super();
 
-    manifestEmitter.addEventListener(AppEvents.manifestUpdate, async () => {
-
-      this.manifest = await getManifest();
+    manifestEmitter.addEventListener(AppEvents.manifestUpdate, async (maniUpdates: any) => {
+      console.log('maniUpdates', maniUpdates);
+      if (maniUpdates) {
+        this.manifest = maniUpdates.detail;
```

Comment:
is this the json of the manifest or the details object from a code mirror manifest update?

Reply:
Its the json of the manifest (or generated manifest) merged with the latest changes